readthedocs/proxito/views/serve.py

```diff
@@ -259,11 +264,17 @@ def get(self, request, proxito_path, template_name='404.html'):
         # If that doesn't work, attempt to serve the 404 of the current version (version_slug)
         # Secondly, try to serve the 404 page for the default version
         # (project.get_default_version())
-        versions = [version_slug]
+        versions = [(version_slug, doc_type)]
         default_version_slug = final_project.get_default_version()
         if default_version_slug != version_slug:
-            versions.append(default_version_slug)
-        for version_slug_404 in versions:
+            doc_type = (
```

Comment:
can we change the name of this variable? we are overriding the original one.

```suggestion
            default_version_doc_type = (
```

Reply:
We don't use the original variable after those lines

Reply:
Yes, but it's confusing. It's safer to use a different name that reflects what it is exactly.